_includes/faq.md

```diff
@@ -0,0 +1,23 @@
+## FAQ
+
+### Why is there no agenda?
+
+We call this "Everybody talks everybody listens"™️.
+Each participant is given a slot to present the subject of their choosing. 
```

Comment:
I'd add a bit more detail after this - something like: "There are no restrictions on the subject of the talk: participants can talk about the biggest challenge they're facing, a cool tool they've used or built, or lessons learned from working on a new team -- or none of the above. Sometimes a participant simply poses a question to the entire group, and lively conversation ensues."

Reply:
Oh I see you talk about that below. Take it or leave it then!

Reply:
I do take it! Incorporated into that other answer